graph/src/components/store/mod.rs

```diff
@@ -246,13 +246,39 @@ impl EntityFilter {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct EntityOrderByChildObject {
```

Comment:
It would be good to have comments describing what this and the other `EntityOrderChild*` structs are and what the fields mean. It's sorta clear from reading the code, but would be good to spell it out.

Reply:
e9544aa1c